assets/oag.css

```diff
@@ -196,6 +197,10 @@
 
 	}
 
+	.label {
+		white-space: normal;
```

Comment:
I would also add `text-align: start;` so that when it wraps it is not aligned in the center.
```suggestion
		text-align: start;
		white-space: normal;

```

Reply:
Added. thanks